docs/modules/ROOT/pages/ds-dash.adoc

```diff
@@ -0,0 +1,92 @@
+= Data Science Plotly Dash (ds-dash)
+
+The project supports generation of Plotly Dash project boilerplate and quick
+installation and integration of it with OpenShift CD pipelines.
+
+== Purpose of this quickstarter
+
+The quickstarter is provisioning a Dash boilerplate ready to be used.
+The package allows easily build a Dash project, using different Python dependencies.
+It contains the basic setup for Docker, Jenkins, SonarQube and OpenShift.
+
+== What files / architecture is generated?
+
+ ├── Jenkinsfile - This file contains Jenkins build configuration settings
+ ├── files
+ │   ├── docker - This folder contains Docker configuration settings
+ │   │   ├── Dockerfile
+ │   │   └── run.sh - This bash script solves issue with permissions for a container user
+ │   └── src
+ │       ├── app.py - This file is the main entry point in the project.
+ │       └── requirements.txt - This file contains a list of required Python modules to run application
+ └── sonar-project.properties - This file contains SonarQube configuration settings
+
+== Frameworks used
+
+* https://docs.python.org/3.7/tutorial/[Python 3.7]
+* https://plot.ly/dash/[Dash]
+
+== Usage - how do you start after you provisioned this quickstarter
+
+The project should be started automatically by OpenShift. Server should be started
+on the port 8080 in the debug mode.
+
+[source,python]
+----
+app.run('0.0.0.0', 8080, debug=True)
+----
+
+To disable a debug mode set debug to *False*.
```

Comment:
I would rather suggest to write instructions how to start this thing using docker. Don't think it makes much sense to install everything locally, when it needs to run in docker later , anyway

Reply:
This was discussed while creating the quickstarter. The agreement was to locally just run python, so no need for any docker when developing locally. Then, it does not mean that if one ends up requiring, for example, especial dependencies that require OS libraries one will need to go over docker. But this is definitely a wide use case (e.g.: requiring especial OS distros, special dependencies and compilation libraries, other python package management tools...). That is why we wanted to keep it simple.